Base/HomagConnect.Base/Documentation/README.md

```diff
@@ -0,0 +1,171 @@
+# HOMAG Connect Base
+
+## Version history
+
+Version   | Date     | Comment 
+----------|----------|---------
+1.0.0     |05.09.2023| First Draft
+
+## Introduction
+
+HOMAG Connect is the central hub for external applications to communicate with HOMAG applications.
+Each partner needs to register with HOMAG and tapio in order to use this communication interface.
+
+### Terms
+
+Term                      | Description
+--------------------------|--------------------------------------------------------------------------
+Homag Connect             | A central gateway for communicating with HOMAG applications. It is deployed worldwide on different regions. The call will automatically be routed to the nearest endpoint / region.
+SAT                       | Subscription Access Token (similar to a PAT (Personal Access Token)).<br>It can be created in the tapio admin UI by the customer; also it can be revoked if needed.<br>This SAT must be created by the end customer and stored inside the partners application for this customer. It is then passed to HOMAG-Connect to authenticate the access.<br>In tapio this is also called `Authorization key`.
+HOMAG subscription id     | Each partner is assigned a unique partner id which owns subscription Id(s). The subscription id must be passed in each call to HOMAG-Connect. It is mainly used to identify the calling application (it is more an information). The authentication is still done with the SAT.
+
+### Partner / developer experience
+
+The partner must register at HOMAG. Then the partner gets an partner id and can then start to call Homag Connect with this partner id and the SAT for the chosen application (MMR Mobile, intelliDivide, etc.), which was created within an tapio subscription.
```

Comment:
do we also describe "non-mmr" endpoints? We have no partner ID in this case

Reply:
As this is not affecting our currently solution it is up to the next team or contributor who is working on something different to add / extend the documentation in this case